### ISSUE TITLE
AB ST flag doesnt spawn in

### ISSUE THREAD

**Issue by thom4466** [chromiecraft/chromiecraft#2710]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [X] 20-29
- [X] 30-39
- [X] 40-49
- [X] 50-59
- [X] 60

### Current Behaviour

No flag will spawn on ST at start of AB
![bBvl4Ax](https://user-images.githubusercontent.com/11156207/147836946-5b35fb2e-65fa-483f-bae0-a3c927eb2fa2.jpeg)

### Expected Blizzlike Behaviour

Flag should spawn in just like all the other flags.

### Steps to reproduce the problem

1. queue for AB
2. go to flag
3. see it doesnt spaw in like others

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/1a75b3ef0d4e7a31d3691cbe22f30af048efa68b

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Shineslippers** [comment]
Thank you for the report. Can't reproduce it on local AC. Did you saw it only once?
![image](https://user-images.githubusercontent.com/85256266/147850738-77141e13-8289-42be-afc6-98a47fcc77be.png)

**Comment by Stump876** [comment]
I also could not reproduce this. Maybe client side issue, or maybe an error in the cross faction BG module?

**Comment by Shineslippers** [comment]
No, it's not mod. Tested issue with crossfaction enabled.

**Comment by Salehyn** [comment]
I could reproduce this, but just precisely once. The first test game I entered, my alliance char didn't have the flag visible, while the horde one did:
![WoWScrnShot_090823_040017](https://github.com/chromiecraft/chromiecraft/assets/134880284/3d98953a-fe2c-45bb-a78a-9845619955ad)
![WoWScrnShot_090823_040050](https://github.com/chromiecraft/chromiecraft/assets/134880284/6037e7bb-79e2-45b9-934c-ca92343d5cbe)
And any new alliance character entering the game also worked fine:
![WoWScrnShot_090823_040203](https://github.com/chromiecraft/chromiecraft/assets/134880284/d079e78d-42e6-4802-b0c4-5727888db64b)

Any subsequent games/characters had no issues whatsoever, even when i changed accounts. I could imagine this is something that might happen the first time you enter AB with your game client, and then never again (otherwise we'd be seeing raging pvpers)
Feels very similar to https://github.com/chromiecraft/chromiecraft/issues/2371 in a sense that the character-non transforming only happened when I entered bg for the first time, and could not reproduce since.

TL;DR it might require a virgin game client (like a fresh download) to reproduce. I'll try to confirm this from my other laptop when I get the chance.
